### DISCUSSION TITLE
How do I prevent thinning of the scroll bar.

### DISCUSSION THREAD

**Discussion by colejohnson66** [amwx/FluentAvalonia#183] [Q&A]
I'm not a fan of "modern" UIs collapsing the scroll bar to be thinner. It makes it so that I can't drag it with my mouse until it fattens itself, which takes a second. Is there a way to disable this?

**Comment by amwx** [comment]
Set the `AllowAutoHide` property to `false` on either ScrollBar or ScrollViewer to prevent them from collapsing
